run/cifs_archive/archive-clips.sh

```diff
@@ -26,71 +28,46 @@ function connectionmonitor {
   done
 }
 
-function moveclips() {
-  ROOT="$1"
-  PATTERN="$2"
-  SUB=$(basename $ROOT)
-
-  if [ ! -d "$ROOT" ]
-  then
-    log "$ROOT does not exist, skipping"
-    return
-  fi
+function syncclips(){
+   ROOT="$1"
+   #log "function syncclips $1 ---> $(find $ROOT -mindepth 1  -type d)"
+   DIR=$(basename $ROOT)
+   mkdir -p $ARCHIVE_MOUNT/$DIR
 
-  while read file_name
-  do
-    if [ -d "$ROOT/$file_name" ]
-    then
-      log "Creating output directory '$SUB/$file_name'"
-      if ! mkdir -p "$ARCHIVE_MOUNT/$SUB/$file_name"
-      then
-        log "Failed to create '$SUB/$file_name', check that archive server is writable and has free space"
-        return
-      fi
-    elif [ -f "$ROOT/$file_name" ]
-    then
-      size=$(stat -c%s "$ROOT/$file_name")
-      if [ $size -lt 100000 ]
-      then
-        log "'$SUB/$file_name' is only $size bytes"
-        rm "$ROOT/$file_name"
-        NUM_FILES_DELETED=$((NUM_FILES_DELETED + 1))
-      else
-        log "Moving '$SUB/$file_name'"
-        outdir=$(dirname "$file_name")
-        if mv -f "$ROOT/$file_name" "$ARCHIVE_MOUNT/$SUB/$outdir"
-        then
-          log "Moved '$SUB/$file_name'"
-          NUM_FILES_MOVED=$((NUM_FILES_MOVED + 1))
-        else
-          log "Failed to move '$SUB/$file_name'"
-          NUM_FILES_FAILED=$((NUM_FILES_FAILED + 1))
-        fi
-      fi
-    else
-      log "$SUB/$file_name not found"
-    fi
-  done <<< $(cd "$ROOT"; find $PATTERN)
+   for i in $(find $ROOT -mindepth 1 -type d ); do
```

Comment:
I was wondering why there is a `for` loop here at all. Rsync can simply sync the parent directory without the need for an external loop, see https://github.com/marcone/teslausb/blob/main-dev/run/rsync_archive/archive-clips.sh

Reply:
Per [the commit author's comment in the corresponding issue](https://github.com/marcone/teslausb/issues/294#issuecomment-542403307) this is so that "every directory is synced separately and deleted afterwards. Not sync all and delete all. I like that a bit better."

But you're right, if the goal is to simplify CIFS copy by using rsync, then just go all the way and use rsync for entire thing.

Reply:
I simplified the solution - now it only uses rsync...

I am not sure, how I should combind all those commits in git? Maybe I have to redo that (which is actually pretty simple )

Reply:
> I am not sure, how I should combind all those commits in git?

I usually use `git rebase -i` to combine changes.

> Maybe I have to redo that (which is actually pretty simple )

That's fine too.

Reply:
just created a new pull-request from a fresh branch.